test/integration/open-with/OpenWith.int.test.js

```diff
@@ -11,7 +11,7 @@ describe('OpenWith', () => {
     });
 
     it('A single integration', () => {
-        cy.route('GET', '**/files/*/open_with_integrations', 'fixture:open-with/integration-adobe');
+        cy.route('GET', '**/files/*/open_with_integrations', 'fixture:open-with/integration-box-edit');
```

Comment:
Looks like you can remove this file then, since its no longer used: https://github.com/box/box-ui-elements/blob/master/test/fixtures/open-with/integration-adobe.json